back-end-side/Controllers/ExtensionMethods.cs

```diff
@@ -13,5 +13,13 @@ public static List<T> MoveToOtherList<T>(this List<T> list, List<T> list2, int e
             }
             return list;
         }
+
+        public static List<Record> RemoveDuplicates(this List<Record> list)
```

Comment:
Is it necessary to return list here? Wouldn't it be simpler to return void and edit ```list``` straightaway? You will be able to use ```RecordsFromFile.RemoveDuplicates();``` instead of ```RecordsFromFile = RecordsFromFile.RemoveDuplicates();```

Reply:
Great idea, will implement it